sw/device/silicon_creator/lib/drivers/retention_sram_functest.c

```diff
@@ -14,12 +14,20 @@
 
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 
+// Variables with of type `retention_sram_t` are static to reduce stack
```

Comment:
nit: s/Variables with of type/Variables of type/

Reply:
Ah, thanks!